browser/brave_vpn/dns/brave_vpn_dns_observer_factory.cc

```diff
@@ -6,13 +6,36 @@
 #include "brave/browser/brave_vpn/dns/brave_vpn_dns_observer_factory.h"
 
 #include <memory>
+#include <string>
+#include <unordered_map>
+
 #include "brave/browser/brave_vpn/dns/brave_vpn_dns_observer_service.h"
 #include "brave/components/brave_vpn/brave_vpn_utils.h"
 #include "chrome/browser/browser_process.h"
+#include "chrome/browser/policy/chrome_browser_policy_connector.h"
 #include "components/keyed_service/content/browser_context_dependency_manager.h"
+#include "components/policy/core/common/cloud/cloud_policy_store.h"
+#include "components/policy/core/common/cloud/machine_level_user_cloud_policy_manager.h"
+#include "components/policy/core/common/cloud/machine_level_user_cloud_policy_store.h"
+#include "components/policy/core/common/policy_map.h"
 
 namespace brave_vpn {
 
+namespace {
+std::string GetDnsPolicyValue(const std::string& name) {
```

Comment:
This is fine for now; but we could name it simply `GetPolicyValue` since it could be used to fetch any values (not just DNS related ones; ex: `policy::key::kDnsOverHttpsMode`)